extract/extract/extractors/zalando.py

```diff
@@ -136,9 +137,18 @@ def get_sustainability(
         List[str]: Ordered `list` of found sustainability labels
     """
 
-    if cluster := beautiful_soup.find("div", attrs={"data-testid": "cluster-certificates"}):
-        labels = cluster.find_all(attrs={"data-testid": "certificate__title"})
-        return sorted(
-            {label_mapping.get(label.string, CertificateType.UNKNOWN) for label in labels}
-        )
+    data = json.loads(
+        beautiful_soup.find("script", {"type": "application/json", "class": "re-1-13"}).get_text()
+    )
+    labels = []
+    for key, item in data.get("graphqlCache", {}).items():
+        for entry in item.get("data", {}).get("product", {}).get("attributeSuperClusters", [{}]):
+            if entry.get("id", "") == "sustainability":
+                for cluster in entry.get("clusters", [{}]):
+                    if cluster.get("sustainabilityClusterKind", "") == "certificates":
+                        for attribute in cluster.get("attributes", [{}]):
+                            labels.append(urllib.parse.unquote(attribute.get("label", "")))
+
+    if labels:
+        return sorted({label_mapping.get(label, CertificateType.UNKNOWN) for label in labels})
```

Comment:
Is there any other way of doing this? It worked for all products but for shoes right?

Reply:
The only location where I was able to find the sustainability information was inside this JSON, which I guess comes from their API. So if we don't want to use this we would need to "debug" splash, because it is no more loading this information into the actual html or switch to playwright. But since this JSON most likely comes from their API the only thing which might change is the html tag class `"class": "re-1-13"` for retrieveing the JSON.

Reply:
I think examples from 14.4 are outdated.